README.md

```diff
@@ -110,13 +110,14 @@ make build
 
 This will build the `go-spacemesh` binary, saving it in the `build/` directory.
 
-To build a binary for a specific architecture directory use:
+On linux or mac you can build a binary for windows using:
```

Comment:
It would be cool to have a builder Dockerfile for cross-compilation to windows with a single cmd.